### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by sshein**
Created an initial draft for the landing page

### CODE REVIEW

File: src/_experimental-design/index.md [hunk 1]
```diff
@@ -8,5 +8,19 @@ draft: true
...
+## Why showcase experimental components?
+Since our community has many people across different contracts and companies building on VA.gov, we often have teams trying out new component designs. However, before this, there was no place for folks to get visibility into what was happening, resulting in team members working on the same problem spaces without being aware that that was the case.
+
+Additionally, we wanted to create a light-weight process where teams building applications could contribute back to our design system.
+
+## Can I use an experimental component in my application?
+Yes. However, if you are using an experimental component, be aware that it hasn’t necessarily been fully validated. If you are using one, please document any research you’ve done on it’s component page for others to see.
+
+## How do I add an experimental component here?
+To get a component into our experimental system, steps 1-4 in [the process written up here](https://design.va.gov/documentation/contributing-to-the-design-system)
```
Comment (mpelzsherman) [inline src/_experimental-design/index.md:25]: I think this sentence is missing a verb?

```suggestion
To get a component into our experimental system, follow steps 1-4 in [the process written up here](https://design.va.gov/documentation/contributing-to-the-design-system)
```

Comment (rtwell) [reply src/_experimental-design/index.md:25]: agreed, we should add "…**follow** steps 1–4 in…"

Comment (sshein) [reply src/_experimental-design/index.md]: thanks, committed the suggestion. Once you approve, Ryan, I'll merge the pull request and then you can always edit it more afterwards if you like

**Review by rtwell — APPROVED** [review]
lgtm

**Pull request merged**